js/ui/interaction.js

```diff
@@ -36,6 +36,7 @@ function Interaction(el) {
     scrollwheel(zoom);
     el.addEventListener('dblclick', ondoubleclick, false);
     window.addEventListener('resize', resize, false);
+    window.addEventListener('keydown', keydown, false);
```

Comment:
This should be bound to `el`, not `window`.

@mourner, can you explain why Leaflet [binds/unbinds on focus/blur](https://github.com/Leaflet/Leaflet/blob/98bf92530f56c0bee1f0764e0934a25db6a6692d/src/map/handler/Map.Keyboard.js#L44-L45)?